pkg/quota/interfaces.go

```diff
@@ -50,13 +50,15 @@ type Evaluator interface {
 	// If true, admission control must perform quota processing for the operation, otherwise it is safe to ignore quota.
 	Handles(operation admission.Attributes) bool
 	// Matches returns true if the specified quota matches the input item
-	Matches(resourceQuota *api.ResourceQuota, item runtime.Object) (bool, error)
+	Matches(resourceQuota *api.ResourceQuota, item runtime.Object, matchPriorityClassScopes bool) (bool, error)
```

Comment:
i dont understand the reason for the bool argument.

the interface shouldnt need to change.

Reply:
i would like to keep all notion of priority out of the evaluator interface.